### ISSUE TITLE
Time picker is not accessible by keyboard

### ISSUE THREAD

**Issue by jannakha** [mlaursen/react-md#193]
I can't select hours/min with keyboard.
I can navigate with tab through hours, but neither Space nor Enter select hour value.
Date picker works (I can select date value with keyboard only).

**Comment by mlaursen** [comment]
Yeah, I didn't plan out what the active interactions should be when tab-navigating times since the material design docs really have little information on Date and Time pickers. So at this point, tabbing actually selects the next time. This was a stupid choice in hindsight. I guess at this point I could make the time picker interactions be:
- Currently selected time: as is (the Clock hand pointing to the time and the primary color circle surrounding the active time)
- Currently focused time: primary color and increased font size
- Enter now selects time instead of tab

Even though the date picker can be navigated fully with the keyboard, I wouldn't say it is a good UX or the right solution for keyboard primary navigation. I think #180 is the end fix here.
